Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py

```diff
@@ -1085,13 +1085,19 @@ def create_nic_command(client: MsGraphClient, args: dict):
 def main():
     params: dict = demisto.params()
     server = params.get('host', 'https://management.azure.com').rstrip('/')
-    tenant = params.get('tenant_id')
-    auth_and_token_url = params.get('auth_id')
-    enc_key = params.get('enc_key')
-    certificate_thumbprint = params.get('certificate_thumbprint')
+    tenant = params.get('cred_tenant_id', {}).get('password') or params.get('tenant_id')
+    if not tenant:
+        raise DemistoException('Token must be provided.')
```

Comment:
Please see AzureFirewall integration for the Exceptions (all exceptions) - make sure it is the same behavior

Reply:
subscription_id, ID and Token are the required ( according to @michal-dagan certificate_thumbprint isn't required in most integration )
https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#supported-authentication-flows-for-microsoft-integrations